frappe/patches/v14_0/set_relative_file_urls.py

```diff
@@ -0,0 +1,24 @@
+import frappe
+from frappe.query_builder.functions import Substring
+
+
+def execute():
+	site_url = frappe.utils.get_url()
```

Comment:
This relies on requests most of the time, so it can be problematic when used in patches. Many sites have different alias for local name and different public-facing URL. :sweat:

Reply:
Maybe we can identify if the path after the domain is "/files" or "/private/files". But we risk messing up the URLs of other frappe instances in the current instance.

---

I haven't written that piece originally. Just copying the logic. Sites may also just get renamed, rendering the old URL invalid. Ideally, this behaviour shouldn't be allowed at all.

Reply:
The intention in this PR is that `get_full_path` shouldn't be doing this stuff.